zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties

```diff
@@ -409,12 +409,12 @@ ascan.custom.warn.disabled	= Input vectors disabled via the Custom Vectors tab
 
 ascan.desc                     = Active scanner, heavily based on the original Paros active scanner, but with additional tests added
 
-ascan.options.anticsrf.label   = Handle anti-CSRF tokens (experimental functionality)
+ascan.options.anticsrf.label   = Handle Anti-CSRF Tokens (Experimental Functionality)
```

Comment:
I think leaving this one as it was and just adding a period probably makes more sense.

Reply:
Personally I see this one as title more than a sentence. I can change it if you think that would be preferable.

Reply:
Hmmm I'm torn (and it seems I missed the "Inject" one). 

I'm thinking for consistency it makes sense for the green parts to be sentences and the blue parts to be titles. I'm open to discussion though:

![image](https://user-images.githubusercontent.com/7570458/64255156-ce49eb80-ceee-11e9-92eb-a5823e2d720a.png)

Reply:
Agreed.

Reply:
Yup. I agree that the "Inject" one should also be a sentence.

The the sake of fluidity, I would also agree that "Handle anti..." should also be a sentence.